src/pages/ReportDetailsPage.js

```diff
@@ -72,16 +71,20 @@ function ReportDetailsPage(props) {
     const canLeaveRoom = useMemo(() => ReportUtils.canLeaveRoom(props.report, !_.isEmpty(policy)), [policy, props.report]);
     const participants = useMemo(() => lodashGet(props.report, 'participantAccountIDs', []), [props.report]);
 
+    const isGroupDMChat = useMemo(() => ReportUtils.isDM(props.report) && participants.length > 1, [props.report, participants.length]);
```

Comment:
NAB but as far as I know, none of these properties need to be memoized. Not sure if there's been a discussion already about when we should be using useMemo, but I think it's only useful for long running/expensive calculations.

Reply:
Yeah, that's true in general for the `useMemo` but I think this is fine as long as correct set of dependency array is passed.